debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -110,7 +110,21 @@ public static ConnectionFactory patternBasedFactory(String urlPattern, Field...
             String url = findAndReplace(urlPattern, props, varsWithDefaults);
             LOGGER.trace("Props: {}", props);
             LOGGER.trace("URL: {}", url);
-            Connection conn = DriverManager.getConnection(url, props);
+            
+            Connection conn = null;
+            String driverClassName = config.getString(JdbcConfiguration.DRIVER);
+            if (driverClassName == null) {
+                conn = DriverManager.getConnection(url, props);
+            } else {
+                try {
+                    @SuppressWarnings("unchecked")
+                    Class<java.sql.Driver> driverClazz = (Class<java.sql.Driver>) Class.forName(driverClassName, true, classloader);
```

Comment:
What happens if `classloader` is null? I think `Class.forName(...)` will generate an exception, and if so you'll have to add an if-block to handle this case.